parm/config/config.resources.nco.static

```diff
@@ -127,6 +127,8 @@ elif [ $step = "anal" ]; then
 
 elif [ $step = "analcalc" ]; then
 
+    export MPICH_MPIIO_HINTS="*:romio_cb_write=disable"
```

Comment:
remind me again why this shouldn't be set site-wide?

Reply:
It slows down the analysis jobs significantly when set in the anal block of WCOSS2.env (the GFSv16 GSI analcalc JJOB sources the "anal" block of WCOSS2.env). Setting this in the analcalc block of config.resources* or config.analcalc were the only ways I could see to set it only for analcalc.

I can reexamine this setting for other jobs in the WCOSS2 `develop` port and set it more appropriately since we own those GSI scripts now. I wanted to be cautious with `dev_v16` since these updates will feed into the `release/gfs.v16.3.0` branch after this PR goes in.

Reply:
Ok. Its fine for high impact work, but we should reexamine they this job needs this and not any other.

Reply:
The [efcs jobs need this setting](https://github.com/NOAA-EMC/global-workflow/blob/dev_v16/env/WCOSS2.env#L127) but that's the only other job that I've come across that needs it on WCOSS2. Will reexamine this setting in the port of `develop` to WCOSS2.